reconstruction.py

```diff
@@ -511,7 +513,8 @@ def reconstruct(self,evrange=(-1,-1,-1)):
                         m = patt.match(name)
                         run = int(m.group(1))
                         event = int(m.group(2))
-                        img_fr = np.array(tf[key])
+                        img_fr = np.array(tf[key]).T
+                        img_fr = img_fr[::-1]                   #structure for h5 copied from ROOT as it was in the past. Unsure if it is correct
```

Comment:
Just to think: Maybe this could be a function (root_flip), just to be easier to understand in the future.